packages/core/src/controllers/RouterController.ts

```diff
@@ -24,6 +24,7 @@ export interface RouterControllerState {
     | 'ChooseAccountName'
     | 'Connect'
     | 'ConnectingExternal'
+    | 'ConnectingFarcaster'
```

Comment:
Why this have separate page?

Reply:
Oh QR, ok

Reply:
QR is not loaded in preview: https://web3modal-laboratory-c59cqvi45-walletconnect1.vercel.app/library/ethers-all-internal/

Reply:
Took at least 10sec but loaded

Reply:
This is Magic latency, don't think we can do anything on our side to make this better

Reply:
Added farcaster logo. We can pre-load, but magic/farcaster fires status calls after farcaster URI request. Thats why I prefer to load when user requests it.